thinkhazard/templates/admin/common.jinja2

```diff
@@ -1,19 +1,11 @@
 {% macro navbar(request) %}
   <div class="admin-navbar container">
     <ul class="nav nav-pills">
-      <li>
-        <a
-          href="{{ request.route_url('admin_index') }}"
-          class="'active' if request.matched_route.name == 'admin_index' else ''">
-          Hazard categories
-         </a>
+      <li role="presentation" class="{{ 'active' if request.matched_route.name == 'admin_index' else '' }}">
```

Comment:
The `else` statement is not required.
See https://github.com/GFDRR/thinkhazard/blob/master/thinkhazard/templates/admin/hazardsets.jinja2#L26